### DISCUSSION TITLE
b站首页开始投放广告，希望能够屏蔽。

### DISCUSSION THREAD

**Discussion by MJSummer** [the1812/Bilibili-Evolved#5013] [功能建议 / Ideas]
![image](https://github.com/user-attachments/assets/87106959-e3c5-4437-b929-9c339b2c631c)

<div data-v-c6402956="" class="palette-button-adcard is-bottom translucent __web-inspector-hide-shortcut__" style="left: 680px;"><picture data-v-c6402956="" class="v-img ad-img"><source srcset="//i0.hdslb.com/bfs/activity-plat/static/20241104/8298d6b0f8f547347b097f0ce035e59c/zaL6eO4fW0.jpg@1400w_896h.avif" type="image/avif" width="800" data-resolution-width="1400"><source srcset="//i0.hdslb.com/bfs/activity-plat/static/20241104/8298d6b0f8f547347b097f0ce035e59c/zaL6eO4fW0.jpg@1400w_896h.webp" type="image/webp" width="800" data-resolution-width="1400"><img src="//i0.hdslb.com/bfs/activity-plat/static/20241104/8298d6b0f8f547347b097f0ce035e59c/zaL6eO4fW0.jpg@1400w_896h" alt="三国谋定天下" loading="eager" onload="" onerror="typeof window.imgOnError === 'function' &amp;&amp; window.imgOnError(this)" width="800" data-resolution-width="1400"></picture><svg data-v-c6402956="" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 24 24" width="24" height="24" class="exit-icon" fill="currentColor"><path d="M4.75075 4.75352C5.14127 4.36299 5.77444 4.36299 6.16496 4.75352L11.9984 10.587L17.8335 4.75188C18.224 4.36135 18.8572 4.36135 19.2477 4.75188C19.6383 5.1424 19.6383 5.77557 19.2477 6.16609L13.4126 12.0012L19.2464 17.835C19.637 18.2255 19.637 18.8587 19.2464 19.2492C18.8559 19.6397 18.2227 19.6397 17.8322 19.2492L11.9984 13.4154L6.16626 19.2476C5.77573 19.6381 5.14257 19.6381 4.75204 19.2476C4.36152 18.857 4.36152 18.2239 4.75204 17.8334L10.5842 12.0012L4.75075 6.16773C4.36022 5.77721 4.36022 5.14404 4.75075 4.75352z" fill="currentColor"></path></svg><div data-v-c6402956="" class="detail-button"> 查看详情 </div></div>

**Comment by MJSummer** [comment]
`<div data-v-c6402956="" class="palette-button-adcard is-bottom translucent __web-inspector-hide-shortcut__" style="left: 680px;"><picture data-v-c6402956="" class="v-img ad-img"><source srcset="//i0.hdslb.com/bfs/activity-plat/static/20241104/8298d6b0f8f547347b097f0ce035e59c/zaL6eO4fW0.jpg@1400w_896h.avif" type="image/avif" width="800" data-resolution-width="1400"><source srcset="//i0.hdslb.com/bfs/activity-plat/static/20241104/8298d6b0f8f547347b097f0ce035e59c/zaL6eO4fW0.jpg@1400w_896h.webp" type="image/webp" width="800" data-resolution-width="1400"><img src="//i0.hdslb.com/bfs/activity-plat/static/20241104/8298d6b0f8f547347b097f0ce035e59c/zaL6eO4fW0.jpg@1400w_896h" alt="三国谋定天下" loading="eager" onload="" onerror="typeof window.imgOnError === 'function' &amp;&amp; window.imgOnError(this)" width="800" data-resolution-width="1400"></picture><svg data-v-c6402956="" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 24 24" width="24" height="24" class="exit-icon" fill="currentColor"><path d="M4.75075 4.75352C5.14127 4.36299 5.77444 4.36299 6.16496 4.75352L11.9984 10.587L17.8335 4.75188C18.224 4.36135 18.8572 4.36135 19.2477 4.75188C19.6383 5.1424 19.6383 5.77557 19.2477 6.16609L13.4126 12.0012L19.2464 17.835C19.637 18.2255 19.637 18.8587 19.2464 19.2492C18.8559 19.6397 18.2227 19.6397 17.8322 19.2492L11.9984 13.4154L6.16626 19.2476C5.77573 19.6381 5.14257 19.6381 4.75204 19.2476C4.36152 18.857 4.36152 18.2239 4.75204 17.8334L10.5842 12.0012L4.75075 6.16773C4.36022 5.77721 4.36022 5.14404 4.75075 4.75352z" fill="currentColor"></path></svg><div data-v-c6402956="" class="detail-button"> 查看详情 </div></div>`

**Comment by Geeyun-JY3** [comment]
请善用搜索，重复：#5001

**Comment by nostphi** [comment]
现在这个广告可以屏蔽了吗

**Reply by MJSummer** [reply]
尚未。
